code/game/objects/items/oddities.dm

```diff
@@ -196,7 +196,7 @@
 
 /obj/item/weapon/oddity/common/teddy
 	name = "teddy bear"
-	desc = "He will there for you, in tough times."
+	desc = "He will be there for you, even through tough times."
```

Comment:
I feel there was more drama with the "in tough times".

In addition, isn't this a reference to the risk of rain item: https://riskofrain.fandom.com/wiki/Tough_Times

So, the "in tough times" would be fitting with the reference itself.

Reply:
thank you for the feedback